packages/manager/modules/pci/src/components/project/users/translations/Messages_fr_FR.json

```diff
@@ -1,5 +1,5 @@
 {
-  "pci_projects_project_users_title": "Users & Roles",
+  "pci_projects_project_users_title": "Utilisateurs",
```

Comment:
Raised the question internally because we are using same entry menu label here:
https://github.com/ovh/manager/blob/b007fd7130cb676ce59336e8ed77ad46d91299bc/packages/manager/modules/pci/src/projects/project/sidebar/sidebar.constant.js#L228-L234

Reply:
Confirmed that we have to update it.